### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by rodricobsanchez**
## Description

- Update README

## Type of change

- This change requires a documentation update

## Checklist:

- [yes] My code follows PEP8 style guide
- [yes] I have removed unnecessary print statements from my code
- [yes] I have made corresponding changes to the documentation if necessary
- [yes] My changes generate no errors
- [yup] No commented-out code
- [ye] Size of pull request kept to a minimum
- [yuppers] Pull request description clearly describes changes made & motivations for said changes

https://trello.com/c/agxeF6Ex/181-documentation-as-a-developer-i-want-to-be-able-to-see-what-the-ds-app-does-so-that-i-can-locally-deploy-and-know-who-built-it

### CODE REVIEW

**Review by RylandGomez — COMMENTED** [review]
Overall great changes and formatting, I just have the one gripe concerning upper management being cataloged in FT 42. I think listing them with their titles at the top of the list is more applicable, since their roles supersede any single team iteration.

**Comment by rodricobsanchez** [comment]
> Overall great changes and formatting, I just have the one gripe concerning upper management being cataloged in FT 42. I think listing them with their titles at the top of the list is more applicable, since their roles supersede any single team iteration.

Please kindly check updated Contributors section I added your suggestion

File: README.md [hunk 1]
```diff
@@ -1 +1,92 @@
...
+
+# Key Features
+
+- Filtration system for connecting mentors with mentees based on skills and experience
+- Use sentiment analysis to rate if a match was successful
+- Filtration system for connecting mentors with mentees based on skills and experience
+- Use sentiment analysis to rate if a match was successful
+- Filtration system for connecting mentors with mentees based on skills and experience
+- Use sentiment analysis to rate if a match was successful
+
```
Comment (RylandGomez) [inline README.md:35]: Content repeated three times

**Review by RylandGomez — CHANGES_REQUESTED** [review]
Reformatting of Contributors looks fantastic! One last thing I did notice was that Key Features repeats its content three times. Fix that and it's an easy approval from me!

File: README.md (hunk 1 above)
Comment (rodricobsanchez) [reply README.md:35]: forgot that lol

Comment (rodricobsanchez) [reply README.md:32]: should be ready for review

**Review by Annapurnaj91 — APPROVED** [review]
Everything looks good for me

**Review by RylandGomez — APPROVED** [review]
Solid README that's expandable in future team iterations and project scope, with a clean layout. Nicely done!

**Review by archana062031 — APPROVED** [review]
Looks good to me!

**Pull request merged**